File: GerenciaMusic360/ClientApp/app/shared/models/contractTerms.ts
import { ITerms } from "./terms";

export interface IContractTerms {
    id: number;
    contractId: number;
    termId: number;
    position: number;

    term?: ITerms;
}

export interface IContractTermType {
    id: number;
    termTypeId: number;
    contractId: number;
    termTypeName: string;
}

File: GerenciaMusic360.Entities/ContractTerms.cs
namespace GerenciaMusic360.Entities
{
    public class ContractTerms
    {
        public int Id { get; set; }
        public int ContractId { get; set; }
        public int TermId { get; set; }
        public short Position { get; set; }
        public int TermTypeId { get; set; }

        public virtual Terms Term { get; set; }
    }
}
